### PR TITLE
Update Russian translations

### PR DESCRIPTION
**Pull request by sbolotovms**

### CODE REVIEW

File: android/assets/jsons/translations/Russian.properties [hunk 1]
```diff
@@ -624,7 +622,7 @@ Convert production to science at a rate of [rate] to 1 = Обмен произв
 The city will not produce anything. = Город ничего не будет производить.
 Worked by [cityName] = Обрабатывается в г. [cityName]
  # Requires translation!
-Lock = 
+Lock = Блокировать
```
Comment (sbolotovms) [inline android/assets/jsons/translations/Russian.properties:625]: I'd like to have more context on this - is it a noun (like a physical lock with a key), a verb (like a "lock city production" or something alike) or something else? Currently translated as a second variant - verb.

File: android/assets/jsons/translations/Russian.properties [hunk 2]
```diff
@@ -935,9 +933,9 @@ Paper Maker = Бумажная мастерская
 Circus = Цирк
 
  # Requires translation!
-Must be next to river = 
+Must be next to river = Должно быть рядом с водой
```
Comment (sbolotovms) [inline android/assets/jsons/translations/Russian.properties:936]: What must be next to river, city?

Comment (sbolotovms) [reply android/assets/jsons/translations/Russian.properties:936]: To provide more context on why I ask it, Russian is a gendered language so depending on the item sentence would look different (e.g. "city" is masculine, "building" is neuter and "farm" is feminine, so it'd be different for these 3).

Comment (yairm210) [reply android/assets/jsons/translations/Russian.properties:936]: Building

File: android/assets/jsons/translations/Russian.properties (hunk 1 above)
Comment (yairm210) [reply android/assets/jsons/translations/Russian.properties:625]: Yup, it's a verb (lock the tile)

**Comment by yairm210** [comment]
merge translations

**Pull request merged**